rules/autoconsent/automattic-cmp-optout.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "automattic-cmp-optout",
+  "prehideSelectors": ["form[class*=\"cookie-banner\"]"],
+  "detectCmp": [{ "exists": "form[class*=\"cookie-banner\"]" }],
```

Comment:
This has a false positive on https://www.bundesfinanzministerium.de/. I think we can add the following to avoid that though:
```suggestion
  "detectCmp": [{ "exists": "form[class*=\"cookie-banner\"][method=\"post\"]" }],
```

Reply:
Adding the "post method" did work on my end.